pages/linux/genid.md

```diff
@@ -0,0 +1,24 @@
+# genid
+
+> Generate IDs, such as snowflakes, UUIDs, and a new GAID.
+> More information: <https://github.com/bleonard252/genid>.
+
+- Generate a UUIDv4:
+
+`genid uuid`
+
+- Generate a UUIDv5 with a namespace UUID and a specific name:
+
+`genid uuidv5 {{{ce598faa-8dd0-49ee-8525-9e24fff71dca}}} {{name}}`
+
+- Generate a Discord Snowflake, without a trailing newline (for scripts):
```

Comment:
```suggestion
- Generate a Discord Snowflake, without a trailing newline (useful in shell scripts):
```

Reply:
I chose to shorten it since the terminal preview looked a bit strange when it overflowed.